sass/_OmniboxResultList.scss

```diff
@@ -14,8 +13,26 @@
   }
 }
 
+.coveo-list-layout,
+.coveo-card-layout,
+.coveo-table-layout {
+  &.CoveoResult {
```

Comment:
Why do we need this `&` if we are setting the mixin on the stand-alone `CoveResult` on line 25? Are the styles being overwritten forcing you to use a more specific selector to override?

If so, do we need the stand-alone on line 25 or can it be removed?

Reply:
Yeah, we need both as it's possible to either have the "layout" manually specified in the page (depending on how the result list is configured), or to have no "layout" classes (if no layout is specified), so we need to cover both cases...